### PR TITLE
add love topic

### PR DESCRIPTION
**Pull request by ezhasyafaat**

### CODE REVIEW

File: config/quotes.php [hunk 1]
```diff
@@ -1429,5 +1429,150 @@
...
+            [
+                'quote' => 'There is never a time or place for true love. It happens accidentally, in a heartbeat, in a single flashing, throbbing moment.',
+                'author' => 'Sarah Dessen',
+            ],
+            [
+                'quote' => 'Love is that condition in which the happiness of another person is essential to your own.',
+                'author' => 'Robert A. Heinlein',
+            ],
+            [
+                'quote' => 'Love never dies a natural death. It dies because we don’t know how to replenish its source. It dies of blindness and errors and betrayals. It dies of illness and wounds; it dies of weariness, of witherings, of tarnishings.',
```
Comment (Kristories) [inline config/quotes.php]: Please replace `’` to `\'` for "don’t", and all worlds contain `’` or a single quote.

File: config/quotes.php [hunk 2]
```diff
@@ -1429,5 +1429,150 @@
...
+            [
+                'quote' => 'I love you not only for what you are, but for what I am when I am with you. I love you not only for what you have made of yourself, but for what you are making of me. I love you for the part of me that you bring out.',
+                'author' => 'Elizabeth Barrett Browning',
+            ],
+            [
+                'quote' => 'If I know what love is, it is because of you.',
+                'author' => 'Hermann Hesse',
+            ],
+            [
+                'quote' => 'I fell in love with her courage, her sincerity, and her flaming self respect. And it’s these things I’d believe in, even if the whole world indulged in wild suspicions that she wasn’t all she should be. I love her and it is the beginning of everything.',
```
Comment (Kristories) [inline config/quotes.php]: "it’s"

Comment (Kristories) [reply config/quotes.php]: "I’d"

Comment (Kristories) [reply config/quotes.php]: wasn’t

File: config/quotes.php [hunk 3]
```diff
@@ -1429,5 +1429,150 @@
...
+                'author' => 'F.Scott Fitzgerald',
+            ],
+            [
+                'quote' => 'I swear I couldn’t love you more than I do right now, and yet I know I will tomorrow.',
```
Comment (Kristories) [inline config/quotes.php]: couldn’t

File: config/quotes.php [hunk 4]
```diff
@@ -1429,5 +1429,150 @@
...
+            [
+                'quote' => 'Love is so short, forgetting is so long.',
+                'author' => 'Pablo Neruda',
+            ],
+            [
+                'quote' => 'Love is a thing that is full of cares and fears.',
+                'author' => 'Ovid',
+            ],
+            [
+                'quote' => 'Love is not love until love’s vulnerable.',
```
Comment (Kristories) [inline config/quotes.php]: love’s

File: config/quotes.php [hunk 5]
```diff
@@ -1429,5 +1429,150 @@
...
+                'author' => 'Theodore Roethke',
+            ],
+            [
+                'quote' => 'You know you’re in love when you can’t fall asleep because reality is finally better than your dreams.',
```
Comment (Kristories) [inline config/quotes.php]: you’re

Comment (Kristories) [reply config/quotes.php]: can’t

File: config/quotes.php [hunk 6]
```diff
@@ -1429,5 +1429,150 @@
...
+                'author' => 'Dr. Seuss',
+            ],
+            [
+                'quote' => 'I love you without knowing how, or when, or from where. I love you simply, without problems or pride: I love you in this way because I do not know any other way of loving but this, in which there is no I or you, so intimate that your hand upon my chest is my hand, so intimate that when I fall asleep your eyes close.',
+                'author' => 'Pablo Neruda',
+            ],
+            [
+                'quote' => 'Love is like the wind, you can’t see it but you can feel it.',
```
Comment (Kristories) [inline config/quotes.php]: can’t

File: config/quotes.php [hunk 7]
```diff
@@ -1429,5 +1429,150 @@
...
+                'author' => 'Nicholas Sparks',
+            ],
+            [
+                'quote' => 'Love is the magician that pulls man out of his own hat.',
+                'author' => 'Ben Hecht',
+            ],
+            [
+                'quote' => 'The best love is the kind that awakens the soul; that makes us reach for more, that plants the fire in our hearts and brings peace to our minds. That’s what I hope to give you forever.',
```
Comment (Kristories) [inline config/quotes.php]: That’s

**Review by Kristories — CHANGES_REQUESTED** [review]

**Comment by ezhasyafaat** [comment]
Please check it again, thanks

**Review by Kristories — APPROVED** [review]

**Comment by Kristories** [comment]
Thanks!

**Pull request merged**